Makefile

```diff
@@ -1,6 +1,14 @@
 start:
+	test -f .env || make local-config
+	test -d Packages || swift package edit PointFree
 	docker-compose up --build
 
+transcripts:
+	git submodule update --init --recursive
+	ln -fn Transcripts/*.swift Sources/pointfreeco/Transcripts
+	git update-index --assume-unchanged Sources/pointfreeco/Transcripts/
+	make xcodeproj
```

Comment:
Calling `make transcripts` puts our repo into private mode:

1. It checks out the private submodule
2. It copies over the Swift files in the root of that repo (and at the moment assumes clobbering **AllEpisodes.swift**)
3. It locally ignores changes in that directory (so we don't accidentally commit them to this public repo)
4. Regenerates the Xcode project